xivo_dao/alchemy/context.py

```diff
@@ -37,7 +37,7 @@ class Context(Base):
 
     id = Column(Integer)
     tenant_uuid = Column(String(36), nullable=False)
-    name = Column(String(39), nullable=False)
+    name = Column(Text, nullable=False)
```

Comment:
Are you sure that asterisk support infinite char?
Don't have objection to increase size, but infinite seem maybe too huge?
Since this key is used as reference by other table, we should also use same constraint, no?

Reply:
I did not find any information on size limit for the name, except coming from our own documentation and our WPEP-006. I asked @pc-m about it and he does not remember why there was such limit.
However, since we autogenerate it and it has a fixed limit based on that autogeneration, I could indeed limit it to the possible maximum we can generate.
For the other table reference, you're right, I forgot to change the schemas in the DAO. I did change the tables in the migration script though, so I saw the issue :laughing:

Reply:
**Update**: found the maximum. It is 80 characters in various call handling functions and in the AST_MAX_CONTEXT variable ([see here](https://github.com/asterisk/asterisk/blob/master/include/asterisk/channel.h#L135)), so that pretty much sums it up. I will then modify everything to make it 80 characters max.